src/view-backend.c

```diff
@@ -70,6 +70,7 @@ wpe_view_backend_destroy(struct wpe_view_backend* backend)
     backend->fullscreen_client_data = NULL;
 
     backend->activity_state = 0;
+    backend->refresh_rate = 0;
```

Comment:
I suppose we could have this initialized to `60`, which is the most typical “standard” value.

Reply:
Right, this is the destructor though? I guess initialisation is down to the implementer, but I don't know what implication that has for adding new values...